node/overseer/overseer-gen/proc-macro/src/impl_builder.rs

```diff
@@ -215,10 +218,10 @@ pub(crate) fn impl_builder(info: &OverseerInfo) -> proc_macro2::TokenStream {
 		// Baggage fields follow subsystems
 		let fname = &bag_field.field_name;
 		let field_type = &bag_field.field_ty;
-		let impl_generics = recollect_without_idx(&baggage_passthrough_generics[..], idx);
+		let impl_generics = recollect_without_idx(&baggage_state_generics[..], idx);
```

Comment:
Actually they are both, `state` and `passthrough`? How about `baggage_passthrough_state_generics`

Reply:
It is a bit too verbose, but more descriptive indeed.